src/caselawclient/models/press_summaries.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     def neutral_citation(self) -> str:
         return get_xpath_match_string(
             self.content_as_xml_tree,
-            "/akn:akomaNtoso/akn:doc/akn:mainBody/akn:p/akn:neutralCitation/text()",
+            "/akn:akomaNtoso/akn:doc/akn:preface/akn:p/akn:neutralCitation/text()",
```

Comment:
Hmmm. Are there issues with moving press summaries because of this difference?

Reply:
discussed offline - nope. we just need to update the `set_document_citation` method to handle updating ncns but we dont need that just yet. There is a card for that in TODO and I do have a branch where I started on it